### PR TITLE
Feature/arrays

### PR DESCRIPTION
**Pull request by jordigraup**

### CODE REVIEW

File: staff/jordi-grau/arrays/array-functions/every.spec.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+describe('every', function () {
+    it('delete the last index position of one array', function () {
```
Comment (alex-martinez-jativa) [inline staff/jordi-grau/arrays/array-functions/every.spec.js:4]: Hi Jordi, this title is wrong, 'every' method tests whether all elements in the array pass the test implemented by the provided function. It returns a Boolean value.

File: staff/jordi-grau/arrays/array-functions/filter.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        var array = [5,2,1];
+
+        var newArray = filter(array, function(element){
+            return element < 4;
+        });
+
+        expect(newArray[0]).toBe(2);
+        expect(newArray[1]).toBe(1);
+    });
+    it('filter a array and return -1 if no find coincidence', function () {
```
Comment (alex-martinez-jativa) [inline staff/jordi-grau/arrays/array-functions/filter.spec.js:14]: the filter method, if don't meet with the condition, will be return an empty array, not -1

**Comment by alex-martinez-jativa** [comment]
Jordi, you should implement more Arroz methods, and implement unhappy paths on your tests

File: staff/jordi-grau/arrays/array-functions/every.spec.js (hunk 1 above)
Comment (jordigraup) [reply staff/jordi-grau/arrays/array-functions/every.spec.js]: Solved!

File: staff/jordi-grau/arrays/array-functions/filter.spec.js (hunk 2 above)
Comment (jordigraup) [reply staff/jordi-grau/arrays/array-functions/filter.spec.js]: Solved

**Pull request merged**